src/bundles/csg/functions.ts

```diff
@@ -10,90 +10,198 @@
 
 /* [Imports] */
 import { primitives } from '@jscad/modeling';
-import { colorize } from '@jscad/modeling/src/colors';
+import { colorize as _colorize } from '@jscad/modeling/src/colors';
 import {
   type BoundingBox,
-  measureArea,
   measureBoundingBox,
-  measureVolume,
 } from '@jscad/modeling/src/measurements';
 import {
   intersect as _intersect,
   subtract as _subtract,
   union as _union,
 } from '@jscad/modeling/src/operations/booleans';
 import { extrudeLinear } from '@jscad/modeling/src/operations/extrusions';
-import {
-  align,
-  center,
-  mirror,
-  rotate as _rotate,
-  scale as _scale,
-  translate as _translate,
-} from '@jscad/modeling/src/operations/transforms';
+import { align } from '@jscad/modeling/src/operations/transforms';
+import { serialize } from '@jscad/stl-serializer';
+import save from 'save-file';
 import { SILVER } from './constants.js';
 import { Core } from './core.js';
 import type { Color, Coordinates, Solid } from './jscad/types.js';
-import { clamp, hexToColor, type RenderGroup, Shape } from './utilities';
+import {
+  hexToColor,
+  type Entity,
+  Shape,
+  Group,
+  type RenderGroup,
+} from './utilities';
+import { type List } from './types';
+
+/**
+ * Colour the shape using the specified hex colour code.
+ *
+ * @param {Shape} shape - The Shape to be coloured and returned
+ * @param {string} hex - The colour code to use
+ * @returns {Shape} The colorized shape
+ */
+function colorize(shape: Shape, hex: string) {
+  let color: Color = hexToColor(hex);
+  let coloredSolid: Solid = _colorize(color, shape.solid);
+  return new Shape(coloredSolid);
+}

```

Comment:
Just like the workflow warning says, this function does not seem to be used anywhere?

Reply:
> Was there any reason functions like `scale_(x|y|z)`, `flip_(x|y|z)`, `transform_(x|y|z)`, `shape_center`, `area`, `volume`, etc. were removed from the list of exports?
It was a change requested by Prof Low to make the set of exported functions less redundant

> Just like the workflow warning says, this function does not seem to be used anywhere?
Thanks for reminding :) Going to remove it in the latest commit